about/html/edit.js

```diff
@@ -78,6 +84,21 @@ exports.create = function (api) {
       else return name.current
     })
 
+
+    var balances_div = h('div.balances')
+
+    mutual.getAccountBalances(id, (error, balances) => {
+      if (balances == null) return ''
+
+      var balance_els = [];
+      Object.keys(balances).forEach(function(key) {
+        console.log(key)
```

Comment:
Maybe remove this?

Reply:
agree - I can't push to this branch tho :C

Reply:
hmmm, pretty sure you can @dangerousbeans ? 
https://github.com/ssbc/patchbay/settings/collaboration

Reply:
or maybe https://github.com/ssbc/patchbay/invitations